### PR TITLE
Feature/users

### PR DESCRIPTION
**Pull request by kamicut**
This PR:
- switches the DB, tests, migrations, to postgres
- Allows logging in with OSM
- Adds some new tests fixtures
- Adds a new schema for users to remove things we can get from the OSM profile

This requires new environment variables for logging in with OSM and the local postgres DB

### CODE REVIEW

**Comment by kamicut** [comment]
I'll have to fix the build so that it works on CI, but could you take a quick look @alyssadelaine @scisco for approach?

File: frontend/src/components/charts/HomeMap.js [hunk 1]
```diff
@@ -15,14 +15,15 @@ class CampaignMap extends Component {
       renderWorldCopies: false,
       interactive: false
     });
+    console.log(this.props.overlay)
```
Comment (scisco) [inline frontend/src/components/charts/HomeMap.js]: could you remove this?

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, {Component} from 'react';
```
Comment (scisco) [inline frontend/src/App.js]: This has linting issues

Comment (kamicut) [reply frontend/src/App.js]: Where did you see the linting error?

**Comment by alyssadelaine** [comment]
@kamicut I think "yarn clocks" should create the database if it hasn't yet been created; right now trying to run the commands in the order in the README gives an error

**Pull request merged**